docs-gen/markdown/rules/FL0028.md

```diff
@@ -14,3 +14,12 @@ Extract code out into private methods or functions.
 
 ## Rule Settings
 
+    [lang=javascript]
+    {
+        "MaxLinesInProperty": {
+            "enabled": false,
+            "config": {
+                "maxLines": 70
+            }
```

Comment:
@duckmatt Json example extracted from `DefaultConfiguration.json`. Is this `config`required? If yes, then other docu pages Needs to be fixed (eg FL0029)

Reply:
On testing with configuration of `maxCharactersOnLine` the `config` seems to be required.

Reply:
Ideally it shouldn't be required and use the default if not specified, I'll take a look

Reply:
tested it with 0.13.3 - didn't test if the rework of #388 Change the behaviour